src/renderer/components/inputs/DirectoryInput.tsx

```diff
@@ -90,7 +93,9 @@ export const DirectoryInput = memo(
                     isDisabled={!displayDirectory}
                     onClick={() => {
                         if (displayDirectory) {
-                            clipboard.writeText(displayDirectory);
+                            ipcRenderer
+                                .invoke('clipboard-writeText', displayDirectory)
+                                .catch(log.error);
```

Comment:
Why were we using electron clipboard in the first place? There's a [browser API for this](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText). Wouldn't that work too?

Reply:
Probably yeah, I can check that

Reply:
Ok, I replaced all the read/write text ones. Can't replace the others as the native clipboard doesn't support as many formats as electron's does.